dotCMS/src/main/java/com/dotcms/rest/api/v1/temp/TempFileAPI.java

```diff
@@ -217,17 +217,20 @@ public DotTempFile createTempFileFromUrl(final String incomingFileName,
    * @return
    * @throws IOException
```

Comment:
remove this, the method is not throwing a IOException anymore

Reply:
done